### DISCUSSION TITLE
Is there any method to check "other" in visualizer dashboard?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1450] [Q&A]
Hi @vincentbernat / expert,

May i know what it means "other" in visualization dashboard? Is it cannot identify the ip address? As i know in netflow it should send with ip address info. Any other way to decrease the info in the "other" info?

Thanks

**Comment by vincentbernat** [comment]
You can increase the limit in the left pane.

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

I'm already put limit 50 as per snapshot below. But i'm still can see "other" not exact info. Appreciate your advise.

![image](https://github.com/user-attachments/assets/670a7035-e9c7-413a-aa6f-a12e7d9603ca)


Thanks

**Reply by kronicklez** [reply]
Hi @vincentbernat / expert,

Any other method that we can reduce the "other" info in visualize sankey? I'm already change the limit to 100 but it still show "other" .

Thanks
